tests/test_general_interface.py

```diff
@@ -1,27 +1,44 @@
 #  Copyright (c) 2021 Robert Lieck
 
 from unittest import TestCase
+from unittest.mock import patch
 
 from pitchtypes import Interval, Chromatic, Diatonic, Pitch
 
 class TestGeneralInterface(TestCase):
 
+    @patch.multiple(Interval, __abstractmethods__=set())
```

Comment:
The `@abstractmethod` decorators in abstract base classes prevents them from being instantiated, so `Interval()` won't work (see [the abc documentation])(https://docs.python.org/3/library/abc.html)). The patch is just for circumventing this check so that we can instantiate `Interval` during testing.